### PR TITLE
Make markshared appear in global modlog

### PR DESCRIPTION
**Pull request by CheeseMuffin**

### CODE REVIEW

File: chat.js [hunk 1]
```diff
@@ -615,11 +615,12 @@ class CommandContext {
...
 	 * @param {string | User} user
 	 * @param {string} note
+	 * @param {object} options
 	 */
-	globalModlog(action, user, note) {
+	globalModlog(action, user, note, options = {}) {
 		let buf = `(${this.room.id}) ${action}: `;
 		if (typeof user === 'string') {
-			buf += `[${toId(user)}]`;
+			buf += `[${options.unfiltered ? user : toId(user)}]`;
```
[the remark this answers is not in the document]
Comment (CheeseMuffin) [reply chat.js]: Yeah only cases it seems to happen where its not already the id is unlocking/unnamelocking/ungbanning where I don't imagine that not using the id is a big deal.

**Comment by Zarel** [comment]
Oh, no, those are still a big deal. Can you just call `toId` before passing to unlock/unnamelock/ungban?

**Comment by CheeseMuffin** [comment]
I updated unbattleban, unlock, and unnamelock - the others use the response from Punishments which should already be the id, I believe (though I can change those if necessary).

**Comment by Zarel** [comment]
Looks good!

**Pull request merged**